cranelift/codegen/src/isa/aarch64/abi.rs

```diff
@@ -320,7 +320,7 @@ impl ABIMachineSpec for AArch64MachineDeps {
                     Some((ty, slot_offset))
                 })
                 .map(|(ty, offset)| ABIArgSlot::Stack {
-                    offset,
+                    offset: i32::try_from(offset).unwrap(),
```

Comment:
We don't currently check that the stack frame size is less than 2GB, so this unwrap can panic: https://github.com/bytecodealliance/wasmtime/issues/7916